FirebaseAppCheck/Sources/Core/FIRAppCheckSettings.h

```diff
@@ -32,7 +34,7 @@ FOUNDATION_EXPORT NSString *const kFIRAppCheckTokenAutoRefreshEnabledInfoPlistKe
 @end
 
 /// Handles storing and updating the Firebase app check wide settings and parameters.
-@interface FIRAppCheckSettings : NSObject <FIRAppCheckSettingsProtocol>
+@interface FIRAppCheckSettings : GACAppCheckSettings <FIRAppCheckSettingsProtocol>
```

Comment:
Alternatively, `FIRAppCheckSettings` could conform to `GACAppCheckSettingsProtocol` and hold an instance of `GACAppCheckSettings`, instead of subclassing from `GACAppCheckSettings`, if we want to avoid inheritance.

This would add a small amount of boilerplate since the implementation of `@property(nonatomic, assign) GACAppCheckTokenAutoRefreshPolicy tokenAutoRefreshPolicy;` would need to call through to the held `GACAppCheckSettings` instance. E.g.:
```
- (GACAppCheckTokenAutoRefreshPolicy)tokenAutoRefreshPolicy {
  return self.appCheckSettings.tokenAutoRefreshPolicy;
}

- (void)setTokenAutoRefreshPolicy:(GACAppCheckTokenAutoRefreshPolicy)tokenAutoRefreshPolicy {
  self.appCheckSettings.tokenAutoRefreshPolicy = tokenAutoRefreshPolicy;
}
```

Reply:
For simplicity, I'll hold off on the alternative approach above unless we find a specific need.